vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -43,6 +43,7 @@ void call(Map args = [:]) {
     sh """
         cp -v ${srcDir}/${baseName}.${extension} ${srcDir}/${baseName}-latest.${extension}
         cp -v ${srcDir}/${baseName}.${extension}.sha512 ${srcDir}/${baseName}-latest.${extension}.sha512
+        cp -v ${srcDir}/../manifest.yml ${srcDir}/${baseName}-latest.${extension}.yml
```

Comment:
Maybe add word manifest in it to make it clear?
```suggestion
        cp -v ${srcDir}/../manifest.yml ${srcDir}/${baseName}-latest.${extension}-manifest.yml
```

Reply:
Added now.